<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordEditWidget</class>
 <widget class="QWidget" name="PasswordEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>78</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="enterPasswordLabel">
     <property name="text">
      <string>Enter password:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="PasswordWidget" name="enterPasswordEdit" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="accessibleName">
      <string>Password field</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="repeatPasswordLabel">
     <property name="text">
      <string>Confirm password:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="PasswordWidget" name="repeatPasswordEdit" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="accessibleName">
      <string>Repeat password field</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordWidget</class>
   <extends>QWidget</extends>
   <header>gui/PasswordWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>enterPasswordEdit</tabstop>
  <tabstop>repeatPasswordEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
